core/src/main/resources/static/js/registerUsers.js

```diff
@@ -426,23 +426,24 @@ app.controller("registerUsersCtrl", function($scope, $http, $location, $window)
                 return;
             }
 
-           var tenantName = "";
-           if($scope.registerUser.tenantName)
-            {
-               var tenantName =  $scope.registerUser.tenantName.trim();
-               if(tenantName.length < 12)
-               {
-                   $scope.alertnote = "Please enter a valid tenant.";
-                   $scope.showAlertToast();
-                   return;
-               }
-            }
+           // var tenantName = "";
+           // if($scope.registerUser.tenantName)
+           //  {
+           //     var tenantName =  $scope.registerUser.tenantName.trim();
+           //     if(tenantName.length < 12)
+           //     {
+           //         $scope.alertnote = "Please enter a valid tenant.";
+           //         $scope.showAlertToast();
+           //         return;
+           //     }
+           //  }
+
             $scope.registrationStarted = 'true';
```

Comment:
i think we can just delete this.

Reply:
registrationStarted variable is used to route the user to thank you page based on its status , it has certain usages in the register.html